tests/test_csc.py

```diff
@@ -5,7 +5,11 @@
 import asyncio
 import salobj
 from lsst.ts.scheduler import SchedulerCSC
-import SALPY_Scheduler
+try:
+    import SALPY_Scheduler
+    run_test = True
+except ImportError:
+    run_test = False
 
```

Comment:
Do you need this change? I'm having a bit of trouble imagining what a user would do with this package if SALPY_Scheduler was not available.

(ts_salobj is a special case because it is quite useful even if nobody ever builds SALPY_Test).